sdk/ml/azure-ai-ml/CHANGELOG.md

```diff
@@ -1,6 +1,12 @@
 # Release History
 
-## 1.1.2 (2022-11-21)
+## 1.2.0 (Unreleased)
+
+### Bugs Fixed
+- Fixed Sweep node not excluding optional input with value `None` in REST object. 
+- Fixed bool test for pipeline expression in download operation.
```

Comment:
pipeline expression -> output

expression is not yet public concept yet.

Reply:
got it, and updated.